src/lib/esnext.promise.d.ts

```diff
@@ -0,0 +1,74 @@
+/** @internal */
```

Comment:
why is this internal? if it was stripped by `stripInternal` (I don't think it is), it would create an invalid declaration file.